### DISCUSSION TITLE
Query regarding usage

### DISCUSSION THREAD

**Discussion by thecodejack** [data-driven-forms/react-forms#997] [Need help] [answered]
Hi

First of all, thanks for this. I am exploring config driven forms and ended up here. Have explored multiple modules and DDF one seems to be complete package. I have a query. I am sorry if this is wrong place to ask same.

Is there any config management to place 2 or more fields in same row? Infact manage complete layout would be awesome using Box(of material ui) or anything that supported by component library

**Comment by rvsia** [comment, marked as answer]
Hello, thank you for your kind words!

If you are using Material UI mapper, you can actually simply place 2 or more fields in the same row.

Each MUI component is wrapped in [Grid component](https://material-ui.com/api/grid/) and you pass props simply by setting: `FormFieldGridProps: { xs: 6 }` etc. You can check a previous issue with example: https://github.com/data-driven-forms/react-forms/issues/871

Box component should be simple to implement if you need it:

```jsx
const FormBox = ({ fields, ...props }) => {
   const { renderForm } = useFormApi();

  return <Box {...props}>{renderForm(fields)}</Box>
}
```

You can check documentation for more details.

(Also moving this to discussions)

**Reply by thecodejack** [reply]
Thanks...I will try this out.

**Reply by Hyperkid123** [reply]
@rvsia since this is a reoccurring question, I think we should add some examples to the docs.

**Reply by Hyperkid123** [reply]
https://data-driven-forms.org/examples/mui-one-row-layout
